src/main/java/uk/gov/hmcts/reform/professionalapi/service/PaymentAccountService.java

```diff
@@ -9,4 +10,6 @@
     Organisation findPaymentAccountsByEmail(String email);
 
     PbaResponse editPaymentAccountsByOrganisation(Organisation organisation, PbaEditRequest pbaEditRequest);
+
+    void deletePaymentsOfOrganization(DeletePbaRequest deletePbaRequest, Organisation organisation);

```

Comment:
_Non-important suggestion_: can we update the name of this method `deletePaymentsOfOrganization` to use the English spelling of the word `Organisation` to keep it in line with the other method names?